<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddQuizModal</class>
 <widget class="QDialog" name="AddQuizModal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="add_quiz_button">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>180</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="header">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>711</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <bold>true</bold>
     <underline>true</underline>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Add Quizes</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="topic_input">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose Topic</string>
   </property>
  </widget>
  <widget class="QLabel" name="question_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Question</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="question_input">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="option1_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Option 1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="option1_input">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="option2_label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Option 2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="option2_input">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="option4_input">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="option4_label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Option 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="option3_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Option 3</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="option3_input">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="option1_radio">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Option 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="correct_option_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Correct Option</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="option2_radio">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>410</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Option 2</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="option3_radio">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>410</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Option 3</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="option4_radio">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>410</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Option 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_question_button">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>480</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save Question</string>
   </property>
   <property name="icon">
    <iconset theme="list-add"/>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="save_topic_button">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>530</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save Topic</string>
   </property>
   <property name="icon">
    <iconset theme="media-floppy"/>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
